purchase_sale_inter_company/models/purchase_order.py

```diff
@@ -115,6 +115,7 @@ def _prepare_sale_order_data(self, name, partner, dest_company,
             'partner_id': partner.id,
             'date_order': self.date_order,
             'auto_purchase_order_id': self.id,
+            'commitment_date': self.date_planned,
```

Comment:
Isn't this change in later onchanges?

Reply:
As long as I tested it doesn't change

Reply:
Well, I prefer to assign it at the end as the previous one for being sure it doesn't change

Reply:
Ok, changed